CodeReady Containers/DetailedStatusViewController.swift

```diff
@@ -40,14 +47,37 @@ class DetailedStatusViewController: NSViewController {
     }
 
     override func viewDidAppear() {
+        self.logs.font = font
+        self.logs.string = "Loading..."
         view.window?.level = .floating
         view.window?.center()
     }
 
     override func viewDidDisappear() {
+        self.timer?.invalidate()
         NotificationCenter.default.removeObserver(self, name: NSNotification.Name(rawValue: "status"), object: nil)
     }
 
+    @objc func updateViewWithLogs() {
+        DispatchQueue.global(qos: .background).async {
+            do {
+                let data = try SendCommandToDaemon(command: Request(command: "logs", args: nil))
```

Comment:
Is this command logged by telemetry?

Reply:
Not yet. There is no telemetry in the tray right now: nothing in the tray, nothing in the daemon.

Reply:
Perhaps better to leave this as this would create a lot of noise (every 2secs). We might have to define an alternative approach for commands like status and logs, as these aren't critical path